framework/Distributions.py

```diff
@@ -1861,7 +1861,8 @@ def initializeDistribution(self):
     else:
       self.xArray   = np.linspace(self.lowerBound,self.upperBound,self.nPoints)
 
-    self.pdfArray = 1/self.xArray.size * np.ones(self.xArray.size)
+    self.pdfArray = 1.0/self.xArray.size * np.ones(self.xArray.size)
+    self.pdfArray = self.pdfArray/np.sum(self.pdfArray)
```

Comment:
I suggest to normalize the self.mapping values in categorical distribution.

Reply:
yes that what I did and i think it fixed the issue